source/_integrations/zwave_me.markdown

```diff
@@ -34,7 +34,33 @@ This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way
 
 {% include integrations/config_flow.md %}
 
-Note: You can use Z-Wave.Me UI with its enhanced diagnostics tools together with the Home Assistant UI.
+{% configuration_basic %}
+
+URL:
+  description: The IP address with the port of the Z-Way server. The IP address can be prefixed with wss:// if HTTPS should be used instead of HTTP (when using find.z-wave.me remote access or public IP with SSL).
+API Token:
+  description: Z-Way API access token of the Z-Way server. To get the token go to the Z-Way user interface Smart Home UI > Menu > Settings > Users > Administrator > API token.
+
+{% endconfiguration_basic %}
+
+When connecting via find.z-wave.me you need to use a token with a global scope (log-in to Z-Way via find.z-wave.me for this).
+
+    Example of connecting to Z-Way in the local network:
+    URL: 192.168.1.39:8083
+    API Token: /112f7a4a-0051-cc2b-3b61-1898181b9950
+
+    Example of connecting to Z-Way via remote access find.z-wave.me:
+    URL: wss://find.z-wave.me
+    API Token: 0481effe8a5c6f757b455babb678dc0e764feae279/112f7a4a-0051-cc2b-3b61-1898181b9950
+
+    Example of connecting to Z-Way with a static public IP address:
+    URL: wss://87.250.250.242:8083
+    API Token: /112f7a4a-0051-cc2b-3b61-1898181b9950
```

Comment:
Have you checked the output of this? Was this as you wanted it to be displayed?

Reply:
Frankly speaking, no. I had no intention to edit it, but Martin suggested to move part of my description changes in core to the web site.

I've checked that only in the markdown editor. Installing the environment is out of my plans. If you can suggest a better way, would appreciate it. Otherwise will check other pages for similar code.

Reply:
> Installing the environment is out of my plans.

Please test it. Thanks.

Reply:
I changed it to something I know how it looks like. Please understand me, I've no intention to spend hours on such a minor improvement. And I don't want you to spend too much time on it too. Hope now it looks ok and will be helpful for users. If you are able to suggest a better layout, I would appreciate it.

Reply:
>  Please understand me, I've no intention to spend hours on such a minor improvement.

Please understand me: We have no intention of accepting any untested piece of code or documentation. If you don't intend to test things, I would recommend not to contribute.

Thanks 👍

Reply:
This was changed two commits ago. Don't you like the current version neither?

Reply:
I did not unresolve the comment, nor did I comment on the contents of the PR. I responded on your comment on how you are approaching this, saying that if you do not want to test, please don't contribute at all. Thanks 👍